### PR TITLE
Release 1.3.0-RC1

### PR DESCRIPTION
**Pull request by lzap**

### CODE REVIEW

File: extras/packaging/rpm/sources/foreman.repo [hunk 1]
```diff
@@ -1,13 +1,13 @@
 [foreman]
-name=Foreman nightly
-baseurl=http://yum.theforeman.org/nightly/$DIST/$basearch
+name=Foreman rc
+baseurl=http://yum.theforeman.org/rc/$DIST/$basearch
 enabled=1
 gpgcheck=0
```
Comment (domcleal) [inline extras/packaging/rpm/sources/foreman.repo]: should be enabled

File: locale/foreman.pot [hunk 2]
```diff
@@ -9,7 +9,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: version 0.0.1\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2013-09-12 12:50+0100\n"
+"POT-Creation-Date: 2013-09-12 14:25+0200\n"
```
Comment (domcleal) [inline locale/foreman.pot]: why the update?  Some strings have been removed...

File: extras/packaging/rpm/sources/foreman.repo (hunk 1 above)
Comment (lzap) [inline extras/packaging/rpm/sources/foreman.repo]: Putting to the Release Process notes ty

File: locale/foreman.pot (hunk 2 above)
Comment (lzap) [inline locale/foreman.pot]: Because it is in the Release Notes, thougth it cannot hurt. So what is wrong here?

Comment (lzap) [inline locale/foreman.pot]: Ah my database was not migrated! Dropping.

**Comment by lzap** [comment]
Let the fun begin!

**Pull request merged**